download/download.go

```diff
@@ -241,7 +246,7 @@ func extractFile(ctx context.Context, input io.Reader, hash hash.Hash, format ar
 					return err
 				}
 			} else {
-				outFl, err := os.Create(path)
+				outFl, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE, fm.Perm())
```

Comment:
This PR looks great. Only thing I'd change is adding the `os.O_TRUNC` flag to match the `os.Create()` function, so just change `os.O_RDWR|os.O_CREATE` to `os.O_RDWR|os.O_CREATE|os.O_TRUNC`.

Reply:
ah yep make sense, truncate the file if it already exists.